diff --git a/simapp/simd/cmd/testnet.go b/simapp/simd/cmd/testnet.go
index 56fffd414..438f5d557 100644
--- a/simapp/simd/cmd/testnet.go
+++ b/simapp/simd/cmd/testnet.go
@@ -587,6 +587,7 @@ func startTestnet(cmd *cobra.Command, args startArgs) error {
 	if _, err := testnet.WaitForHeight(1); err != nil {
 		return err
 	}
+  for {}
 	cmd.Println("press the Enter Key to terminate")
 	if _, err := fmt.Scanln(); err != nil { // wait for Enter Key
 		return err
